integration-tests/src/tests/client/features.rs

```diff
@@ -4,3 +4,5 @@
 mod account_id_in_function_call_permission;
 #[cfg(feature = "protocol_feature_fix_contract_loading_cost")]
 mod fix_contract_loading_cost;
+mod restore_receipts_after_fix;
```

Comment:
Let's maybe call all modules *exactly* as the feature is called? So `restore_receipts_after_fix_apply_chunks`. A mouthful, but no need to thing about how to shorten the name, and might some day serendipitously end up in some grep.